### PR TITLE
Feature/iops 1623 patient

### PR DESCRIPTION
**Pull request by digitalhealthmartin**

### CODE REVIEW

**Comment by digitalhealthmartin** [comment]
Removed unrelated files.

**Review by KazeemHamzat — COMMENTED** [review]
Clarifications on postcode for example patients vs GDPR

**Review by SBajanki1 — DISMISSED** [review]

File: examples/England-Patient-FredricaSmith-Example.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        <name>
+            <family value="SMITH" />
+            <given value="FRANCESCA" />
+            <prefix value="MRS" />
+        </name>
+        <address>
+            <use value="home" />
+            <city value="MALTON" />
+            <district value="NORTH YORKSHIRE" />
+            <postalCode value="YO32 5TT" />
```
[the remark this answers is not in the document]
Comment (digitalhealthmartin) [reply examples/England-Patient-FredricaSmith-Example.xml:43]: Good point but no guidance exists currently and can be picked up in future improvement review

**Comment by digitalhealthmartin** [comment]
Added CS

**Comment by digitalhealthmartin** [comment]
**OLD PR. DO NOT MERGE. CLOSING PR.**

**Pull request closed without merging**